docs/source/matlab/symktensor.rst

```diff
@@ -0,0 +1,8 @@
+``symktensor``
```

Comment:
IMO I think it might be clearer to just remove the placeholders. I'm not sure what the plan is for actually covering these other classes.